```
       >       |  ^~~~~
       > In file included from /build/source/src/wtf/pch.h:41:
       > /build/source/src/wtf/kvm_backend.h:291:26: error: field 'Lapic_' has incomplete type 'kvm_lapic_state'
       >   291 |   struct kvm_lapic_state Lapic_;
       >       |                          ^~~~~~
       > /build/source/src/wtf/kvm_backend.h:291:10: note: forward declaration of 'struct kvm_lapic_state'
       >   291 |   struct kvm_lapic_state Lapic_;
       >       |          ^~~~~~~~~~~~~~~
       > ninja: build stopped: subcommand failed.
       For full logs, run 'nix log /nix/store/qvqlx5jyzm9y2zghazf8nizbqajk6jb3-wtf-0.5.7.drv'.

```
I guess it does not compile for `aarch64` :thinking:

--- a/pkgs/by-name/wt/wtf/package.nix
+++ b/pkgs/by-name/wt/wtf/package.nix
@@ -43,7 +43,7 @@ stdenv.mkDerivation (finalAttrs: {
     homepage = "https://github.com/0vercl0k/wtf";
     changelog = "https://github.com/0vercl0k/wtf/releases/tag/v${finalAttrs.version}";
     license = lib.licenses.mit;
-    platforms = lib.platforms.all;
+    platforms = with lib.platforms; [ "x86_64-linux" ];
     maintainers = with lib.maintainers; [ mikehorn ];
     mainProgram = "wtf";
   };

